src/components/ContactUs/SocialMedia.js

```diff
@@ -14,6 +14,7 @@ export const SocialIcon = (props) => {
       href={props.socialLink}
       target="_blank"
       rel="noreferrer"
+      rel="nofollow"
```

Comment:
We can not have multiple attributes on a JSX element, Please eliminate one of the two attributes of rel.

Reply:
I have removed the "noreferrer' element.

Reply:
But I'm not seeing any new commit to this PR, Please check well and make sure it's removed and push again to this very pr

Reply:
Okay sure

Reply:
I have changed the attributed, Please check it.

![image](https://github.com/KimeraMoses/dav-safaris/assets/110896205/f8ad5f82-24bf-49d3-866b-b953e7b5d72c)